modules/internal/ChapelDomain.chpl

```diff
@@ -1895,9 +1889,6 @@ module ChapelDomain {
     proc ref bulkAdd(inds: [] _value.rank*_value.idxType,
         dataSorted=false, isUnique=false, preserveInds=true, addOn=nilLocale)
         where this.isSparse() && _value.rank>1 {
-      if chpl_warnUnstable then compilerWarning(
-        "bulkAdd() is subject to change in the future.");
```

Comment:
Oops, looks like you removed this warning but didn't attach `@unstable` to the function